laravel/public/css/framework.css

```diff
@@ -266,7 +266,7 @@
  * Forms
  */
  div.form-group {
-
+    padding-left: 0px; /* Remove left pad so form groups sit even on left side w/ other elements */
```

Comment:
So the form groups aren't set slightly to the right of other elements (ie. the index view search bars and h1 tags are now even